efi-no-runtime/efinoruntime.csproj

```diff
@@ -35,4 +39,62 @@
     </ItemGroup>
   </Target>
 
+  <PropertyGroup>
+    <IlcExePath>$(Pkgruntime_win-x64_microsoft_dotnet_ilcompiler)/tools/ilc.exe</IlcExePath>
+    <LinkPath>link</LinkPath>
```

Comment:
did you maybe know if there is a nuget package that contains the `link.exe` that would help for running the sample a lot. Currently you need visual studio installed and a windows plattform to build this UEFI stuff.